tests/web/ClickDemo.cc

```diff
@@ -0,0 +1,107 @@
+//  This file is part of Empirical, https://github.com/devosoft/Empirical
```

Comment:
I think this file has been superseded by the other one?

Reply:
I think the other one shows our thought process more and this one is a more "clean" example. Do you think we should keep both or choose one?

Reply:
keeping both!